server/pulp/server/async/tasks.py

```diff
@@ -344,12 +348,15 @@ def apply_async(self, *args, **kwargs):
         :param tags:        A list of tags (strings) to place onto the task, used for searching for
                             tasks by tag
         :type  tags:        list
+        :param group_id:    The id that identifies which group of tasks a task belongs to
+        :type group_id:     basestring
         :return:            An AsyncResult instance as returned by Celery's apply_async
         :rtype:             celery.result.AsyncResult
         """
         routing_key = kwargs.get('routing_key',
                                  defaults.NAMESPACES['CELERY']['DEFAULT_ROUTING_KEY'].default)
         tag_list = kwargs.pop('tags', [])
+        group_id = kwargs.pop('group_id', None)
```

Comment:
Consider also adding an `isinstance` check that will raise an exception if this argument is not None and is not an instance of `uuid.UUID`.